package.json

```diff
@@ -19,5 +19,8 @@
   "bugs": {
     "url": "http://github.com/jriecken/sat-js/issues"
   },
-  "main": "SAT.js"
+  "main": "SAT.js",
```

Comment:
Can you add a:

```
"scripts": {
  "test": "mocha test"
}
```

section so that the tests can be run just by running `npm test`?